deltachat-ios/Helper/NotificationManager.swift

```diff
@@ -78,7 +84,7 @@ public class NotificationManager {
 
     private func initMsgsNoticedObserver() {
         msgsNoticedObserver =  NotificationCenter.default.addObserver(
-            forName: dcNotificationIncoming,
+            forName: dcMsgsNoticed,
```

Comment:
ahh: _that_ explains the missing updated probably even better than https://github.com/deltachat/deltachat-ios/pull/1119#issuecomment-811184247